packages/x-components/package.json

```diff
@@ -79,23 +79,25 @@
     "@cypress/vue": "~2.2.4",
     "@cypress/webpack-dev-server": "~1.8.4",
     "@empathyco/x-adapter-platform": "^1.0.0-alpha.30",
+    "@empathyco/x-tailwindcss": "^0.2.0-alpha.9",
     "@microsoft/api-documenter": "~7.15.3",
     "@microsoft/api-extractor": "~7.19.4",
     "@rollup/plugin-commonjs": "~21.0.1",
     "@testing-library/jest-dom": "~5.11.1",
-    "@types/autoprefixer": "~9.6.1",
+    "@types/autoprefixer": "~10.2.0",
     "@types/jest": "~27.0.3",
     "@types/node": "~16.4.12",
+    "@types/tailwindcss": "~3.0.10",
     "@types/testing-library__jest-dom": "~5.9.1",
     "@vue/cli-plugin-e2e-cypress": "~5.0.4",
     "@vue/cli-plugin-typescript": "~5.0.4",
     "@vue/cli-plugin-vuex": "~5.0.4",
     "@vue/cli-service": "~5.0.4",
     "@vue/test-utils": "~1.0.3",
     "@vue/vue2-jest": "~27.0.0-alpha.3",
-    "autoprefixer": "~9.7.4",
+    "autoprefixer": "~10.4.4",
     "cypress": "~8.4.0",
-    "cypress-cucumber-preprocessor": "~4.0.0",
+    "cypress-cucumber-preprocessor": "^4.3.1",
```

Comment:
Why has this been updated? and why with a minor version?

Reply:
I honestly don't know, I think is just a mistake. I've reverted it.